2024/regex/dashboard/src/data/reader.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+
+def load_json(file_path: str) -> pd.DataFrame:
+    """
+    Reads a JSON file and creates a pandas DataFrame.
+
+    Parameters:
+    - file_path (str): The path to the JSON file.
+
+    Returns:
+    - pd.DataFrame: The DataFrame created from the JSON file.
+    """
+    try:
+        # Read the JSON file into a DataFrame
+        df = pd.read_json(file_path)
+        return df
+    except Exception as e:
```

Comment:
Does it make sense to catch all exceptions here? It's typically not a good practice in Python (and I give off about it on the channel as well from time to time :) ).

Reply:
Ah, yes that I agree is not good practice. I will change it! Good catch :D